azext_edge/edge/providers/rpsaas/adr/helpers.py

```diff
@@ -0,0 +1,40 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from typing import Dict, Optional
+from ...orchestration.resources import Instances
+
+logger = get_logger(__name__)
+
+
+def check_cluster_connectivity(cmd, resource: dict):
```

Comment:
nit `resource` is specifically an instance resource.

Reply:
whoops - have been using non instance resources (it works)
![image](https://github.com/user-attachments/assets/5bc8709e-de4a-4147-933f-3f2c908fde68)
![image](https://github.com/user-attachments/assets/fe455837-fa1a-44cd-8f3c-f61df80f051c)


used resource as a general "asset/endpoint/whatever else needs this" but if there is a better word, I can change it

Reply:
ok yea makes sense anything with the extended location property, name could be the same or have an rpaas/rpsaas in there but at least we can add a docstring that indicates that rpaas/rpsaas is expected.

Reply:
I do like doc strings (trying to keep the thumbs up as a "I did this but did not push yet" tracker)